deb-arm/Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN grep -v return /root/.bashrc >> /root/newbashrc && cp /root/newbashrc /root/
 RUN apt-get update && apt-get install -y fakeroot curl git procps bzip2 \
     build-essential pkg-config tar libsystemd-dev libkrb5-dev \
     gettext libtool autopoint autoconf libtool-bin \
-    selinux-basics default-jre flex
+    selinux-basics default-jre flex libssl-dev
```

Comment:
This is required to have it work on arm64. It was actually installed before by the `rvm requirements` step, and is installed in the `deb-x64` images